### PR TITLE
pytorch 1.12 successfully install

### PR DESCRIPTION
**Pull request by wanglumeng**
modified some code to pytorch 1.11+ version
tested on Nvidia 3070ti with 470 driver and cuda 11.2/pytorch 1.12
maybe it's a solution for #25

### CODE REVIEW

**Comment by RoyYang0714** [comment]
Dear [wanglumeng](https://github.com/wanglumeng), thanks for the solution! It is indeed that we can successfully install everything on the newer machine and we are testing on RTX 3090 with Cuda 11.3 and Pytorch 1.12 now. I think some parts of the code will also need to be updated for newer PyTorch and some packages will also need to be installed as a newer version. We are working on the patch and will merge the PR after checking the final numbers. If you already have the modified code and the updated requirement files, please let us know. Cheers.

**Review by RoyYang0714 — APPROVED** [review]

**Comment by RoyYang0714** [comment]
I have validated the score on nuScenes validation set and will update the code with a patch on a new commit.
